Doc/extending/newtypes_tutorial.rst

```diff
@@ -255,7 +255,7 @@ allocation and deallocation.  At a minimum, we need a deallocation method::
        CustomObject *self = (CustomObject *) op;
        Py_XDECREF(self->first);
        Py_XDECREF(self->last);
-       Py_TYPE(self)->tp_free(op);
+       Py_TYPE(self)->tp_free(self);
```

Comment:
Shouldn't that be `op`?

Reply:
`tp_free` takes a `void *` so it doesn't matter here. I think it's easier for a newcomer to have `Py_TYPE(X)->tp_free(X)` instead of two names in the call.